lib/svg-preview-view.js

```diff
@@ -27,6 +27,17 @@ class SvgPreviewView extends View {
             this.text('transparent')
           })
         })
+        this.div({ class: 'image-controls-group btn-group' }, () => {
+          this.a({ class: 'btn', outlet: 'zoomOutButton' }, () => {
+            this.text('-')
+          })
+          this.a({ class: 'btn reset-zoom-button', outlet: 'resetZoomButton' }, () => {
+            this.text('100%')
```

Comment:
Of cause this value needs to be adjusted, when ever the user zooms in or out.


You could add this to `zoomTo(zoomValue)`:

```
this.resetZoomButton.text(`${Math.round(zoomValue * 100)}%`)
```